src/Collector/ContainerInterfaceProxy.php

```diff
@@ -163,10 +176,11 @@ private function getServiceProxyFromArray(object $instance, array $params): ?obj
                     try {
                         $params[$index] = $this->get($param);
                     } catch (Exception) {
-                        //leave as is
+                        // leave as is
                     }
                 }
             }
+            /** @psalm-suppress MixedMethodCall */
             return new $proxyClass($instance, ...$params);
         } catch (Exception) {
             return null;
```

Comment:
Does that work with `ObjectProxy` return type?

Reply:
This method returns `?object`